mtgjson5/classes/mtgjson_set.py

```diff
@@ -19,6 +19,7 @@ class MtgjsonSetObject:
     cards: List[MtgjsonCardObject]
     code: str
     code_v3: str
+    printed_languages: List[str]
```

Comment:
I don't know how true this is, as some single cards might have a foreign print while the other cards don't